reader-api/src/test/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# Test configuration for reader-api
+%test.quarkus.http.test-port=9081
+
+# Disable observability in tests to avoid heavy containers
+%test.quarkus.otel.enabled=false
+%test.quarkus.observability-devservices.enabled=false
```

Comment:
Use the `quarkus.observability.enabled=false` setting to disable the observability dev services instead of the `quarkus.otel.enabled` property and the `quarkus.observability-devservices.enabled` properties.